Standardizing creating a (messaging channel) articles (#3235)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/sms/campaign/create.md

> 

Reviewer: Couldn't comment directly on the title because it is not shown in Github edits, but updating "Creating a SMS campaign" nav_title to "Creating an SMS" in the nav bar for consistency.

## Step 1: Choose where to build your message

> Not sure whether your message should be send using a campaign or a Canvas? Campaigns are better for single, simple messaging campaigns, while Canvases are better for multi-step user journeys.

Reviewer: ```suggestion
Not sure whether your message should be sent using a campaign or a Canvas? Campaigns are better for single, simple messaging campaigns, while Canvases are better for multi-step user journeys.
```

6. Select a [subscription group]({{site.baseurl}}/user_guide/message_building_by_channel/sms/sms_subscription_group/) to ensure you’re sending your message to the proper users. When selecting a subscription group, Braze will automatically add a segmenting filter, ensuring that only users subscribed will receive the campaign. Only long codes and short codes that belong to that subscription group will be used to send SMS to target users.

{% alert tip %}
If all of the messages in your campaign are going to be similar or have the same content, compose your message before adding additional variants. You can then choose **Copy from Variant** from the **Add Variant** dropdown.
{% endalert %}

{% endtab %}
{% tab Canvas %}

**Steps:**

1. [Create your Canvas]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas/) using the Canvas wizard.
2. After you've set up your Canvas, add a step in the Canvas builder. Name your step something clear and meaningful.
3. Choose a [step schedule]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/time_based_canvas/#schedule-delay) and specify a delay as needed.

> 4. Filter your audience for this step, as necessary. You can further refine the recipients of this step by specifying segments and adding additional filters. Audience options will be checked after the delay, at the time messages are sent.

Reviewer 2: ```suggestion
4. Filter your audience for this step as necessary. You can further refine the recipients of this step by specifying segments and adding additional filters. Audience options will be checked after the delay at the time messages are sent.
```

{% alert tip %}
{% raw %}
If you plan to use Liquid, be sure to include a default value for your chosen personalization so, in the event your user profile of the recipient is incomplete, they will not receive a blank placeholder `Hi, !`, instead of their name or a coherent sentence.
{% endraw %}
{% endalert %}

### Contact Cards

If desired, you can add a Contact Card to your SMS message to make it easy for your customers to add your business and contact information to their contacts. You can assign common properties to these cards, such as your company’s name, phone number, address, email, and a small photo. Check out [Contact Cards]({{site.baseurl}}/user_guide/message_building_by_channel/sms/mms/contact_card/) to learn more.

## Step 3: Preview and test your message

Braze always recommends previewing and testing your message before sending. Switch to the **Test** tab to send a test SMS to [content test groups]({{site.baseurl}}/user_guide/administrative/app_settings/developer_console/internal_groups_tab/#content-test-groups) or individual users, or preview the message as a user directly in Braze.

![Test SMS]({% image_buster /assets/img/sms_campaign_test.png %})

{% alert tip %}

> If you'd like to test how many segments yout SMS may be split into, test your copy length with our [SMS segment calculator]({{site.baseurl}}/user_guide/message_building_by_channel/sms/campaign/segments/#segment-calculator).

Reviewer: ```suggestion
If you'd like to test how many segments your SMS may be split into, test your copy length with our [SMS segment calculator]({{site.baseurl}}/user_guide/message_building_by_channel/sms/campaign/segments/#segment-calculator).
```

{% endalert %}

## Step 4: Build the remainder of your campaign or Canvas

{% tabs %}
{% tab Campaign %}

> Build the remainder of your campaign; see the sections below for further details on how to best utilize our tools to build SMS messages.

Reviewer 2: ```suggestion
Next, build the remainder of your campaign. See the following sections for further details on how to best utilize our tools to build SMS messages.
```

{% alert tip %}
Interesting in SMS retargeting? Visit our SMS [retargeting article]({{site.baseurl}}/user_guide/message_building_by_channel/sms/campaign/retargeting/) to learn more. 
{% endalert %}

#### Choose conversion events

Braze allows you to track how often users perform specific actions, [conversion events]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/conversion_events/), after receiving a campaign. You have the option of allowing up to a 30-day window during which a conversion will be counted if the user takes the specified action.

> Conversion events help you measure the success of your campaign, for example:

Reviewer 2: ```suggestion
Conversion events help you measure the success of your campaign. For example:
```
